test/e2e/user_management_test.go

```diff
@@ -590,7 +590,7 @@ func (s *userManagementTestSuite) TestUserBanning() {
 		t.Logf("user signup '%s' created", userSignup.Name)
 
 		// Check the UserSignup is created
-		userSignup, err = hostAwait.WaitForUserSignup(t, userSignup.Name)
+		userSignup, err = hostAwait.WaitForUserSignup(t, userSignup.Name, wait.UntilUserSignupHasStateLabel(toolchainv1alpha1.UserSignupStateLabelValueBanned))
```

Comment:
wait for Label to avoid flakiness
Since in the following lines, the test checks that usersignup now has label. 
PR failure - https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/codeready-toolchain_toolchain-e2e/1043/pull-ci-codeready-toolchain-toolchain-e2e-master-e2e/1841349548585259008 - but was unable to produce locally, so added a wait for label.

Reply:
It makes sense, however we don't need lines 597-599 anymore.

Reply:
oh ok, removed the following lines in [cfa49c2](https://github.com/codeready-toolchain/toolchain-e2e/pull/1043/commits/cfa49c266969389c217352754504a3e450b8a1c6)
The wait for Space and Spacebinding to be deleted is still needed right?

Reply:
Yes. Thanks.